pages/api/companies.ts

```diff
@@ -1,5 +1,134 @@
+import { PrismaClient } from '@prisma/client';
 import type { NextApiRequest, NextApiResponse } from 'next';
+import { CompanyData, CompanyListData, CompanyPostData } from '../../types/company';
+import { isEmpty, isValidUrl } from '../../utils/strings/validations';
+import { capitalizeEveryWord, removeProtocolAndWwwIfPresent } from '../../utils/strings/formatters';
+import { createJsonResponse, HttpMethod, HttpStatus, rejectHttpMethod } from '../../utils/http/httpHelpers';
+import { withAuth } from '../../utils/auth/jwtHelpers';
+import { ApiResponse, StatusMessageType } from '../../types/apiResponse';
+import { Nullable } from '../../types/utils';
+import { withPrismaErrorHandling } from '../../utils/prisma/prismaHelpers';
 
-export default function handler(req: NextApiRequest, res: NextApiResponse) {
-  res.status(200).end(`Request sent to ${req.url}\n`);
+enum MessageType {
+  COMPANY_ALREADY_EXISTS,
+  COMPANY_CREATED_SUCCESSFULLY,
+  EMPTY_NAME,
+  INVALID_COMPANY_URL,
+  MISSING_NAME,
+  MISSING_COMPANY_URL,
 }
+
+const prisma = new PrismaClient();
+
+const messages = Object.freeze({
+  [MessageType.COMPANY_ALREADY_EXISTS]: { type: StatusMessageType.SUCCESS, message: 'Company already exists.' },
```

Comment:
```suggestion
  [MessageType.COMPANY_ALREADY_EXISTS]: { type: StatusMessageType.ERROR, message: 'Company already exists.' },
```

Reply:
Not sure if this is a success or error

Reply:
It's a success because if companies are not verified, users will attempt to create them multiple times and that should be okay.